src/index.css

```diff
@@ -0,0 +1,18 @@
+/* body {
```

Comment:
Required
- Please remove any unused code for a cleaner code base and more optimized running app